shell/platform/embedder/embedder.h

```diff
@@ -449,6 +459,13 @@ typedef uint32_t (*UIntFrameInfoCallback)(
     void* /* user data */,
     const FlutterFrameInfo* /* frame info */);
 
+/// Callback for when a frame buffer object is requested with necessary
+/// information for partial repaint.
+typedef void (*FlutterFrameBufferWithDamageCallback)(
+    void* /* user data */,
+    const intptr_t /* fbo id */,
+    FlutterDamage* /* existing damage */);
```

Comment:
@chinmaygarde  this is an out parameter, the embedder populates this struct, but it has a list of rects in it. Do you have any recommendations of how the embedder can supply a buffer for the engine to fill out?